tests/ltp/Makefile

```diff
@@ -144,6 +167,9 @@ clean:
 	sudo rm -rf $(APPDIR) ext2fs appdir $(UPPERDIR) $(WORKDIR) $(HOSTFS)
 	$(MAKE) del_host_users_and_groups
 
+native-clean:
+	sudo rm -rf native_appdir ext2fs
+
```

Comment:
Is this line not needed in native-clean because you are adding host users and groups in native_build $(MAKE) del_host_users_and_groups

Reply:
Removed the addition of host user and groups from these targets since they are run natively